dask/base.py

```diff
@@ -863,6 +863,9 @@ def get_scheduler(get=None, scheduler=None, collections=None, cls=None):
         elif scheduler.lower() in ('dask.distributed', 'distributed'):
             from distributed.worker import get_client
             return get_client().get
+        elif scheduler.lower() in ['processes', 'multiprocessing']:
+            raise ValueError("A compatible version of ``cloudpickle`` is "
+                             "required to use the ``%s`` scheduler." % scheduler)
```

Comment:
Looks good.  I suggest using a Python quote character rather than backticks (otherwise the occasional user may copy-paste and become very confused).  I also recommend dropping the "compatible version" language and saying instead 

`"Please install cloudpickle to use the '%s' scheduler." % scheduler`

We're pretty robust to old cloudpickle versions and mentioning compatible version might lead novice users to wonder what is and isn't ok to install.

(sorry for the nitpicky concerns here, I read a lot of issues from novices and they've probably made me hyper-sensitive to this stuff )

Reply:
Totally fair, this error message is really to help the users and should be well thought out.